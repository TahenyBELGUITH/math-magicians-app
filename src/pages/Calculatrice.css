.d-flex {
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.container-calc {
  width: 70%;
  margin: auto;
}

.inner {
  background-color: rgb(30, 130, 30);
  border: 3px solid burlywood;
  width: 47%;
  padding: 3rem;
  color: #fff;
}
