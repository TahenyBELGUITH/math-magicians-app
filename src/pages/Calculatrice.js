import React from 'react';
import Calculator from '../components/Calculator';
import './Calculatrice.css';

export default function Calculatrice() {
  return (
    <div className="container-calc d-flex">
      <div className="inner">
        <h1>Let&apos;s do Maths !</h1>
      </div>
      <Calculator />
    </div>
  );
}
