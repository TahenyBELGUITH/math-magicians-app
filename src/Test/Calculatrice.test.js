import React from 'react';
import renderer from 'react-test-renderer';
import Calculatrice from '../pages/Calculatrice';

it('it should match the calculator page', () => {
  const tree = renderer.create(<Calculatrice />).toJSON();
  expect(tree).toMatchSnapshot();
});
