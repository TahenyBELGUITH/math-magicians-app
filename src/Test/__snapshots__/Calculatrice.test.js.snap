// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should match the calculator page 1`] = `
<div
  className="container-calc d-flex"
>
  <div
    className="inner"
  >
    <h1>
      Let's do Maths !
    </h1>
  </div>
  <div>
    <div
      className="calculator-container"
    >
      <span>
        Tahtouha
      </span>
      <div
        className="output"
      >
        <div
          className="operation"
        >
          0
           
        </div>
      </div>
      <div
        className="buttons"
      >
        <div
          className="group1"
        >
          <button
            name="AC"
            onClick={[Function]}
            type="button"
          >
            AC
          </button>
          <button
            name="+/-"
            onClick={[Function]}
            type="button"
          >
            +/-
          </button>
          <button
            name="%"
            onClick={[Function]}
            type="button"
          >
            %
          </button>
          <button
            name="+"
            onClick={[Function]}
            type="button"
          >
            +
          </button>
        </div>
        <div
          className="group2"
        >
          <button
            name="7"
            onClick={[Function]}
            type="button"
          >
            7
          </button>
          <button
            name="8"
            onClick={[Function]}
            type="button"
          >
            8
          </button>
          <button
            name="9"
            onClick={[Function]}
            type="button"
          >
            9
          </button>
          <button
            name="รท"
            onClick={[Function]}
            type="button"
          >
            รท
          </button>
        </div>
        <div
          className="group3"
        >
          <button
            name="4"
            onClick={[Function]}
            type="button"
          >
            4
          </button>
          <button
            name="5"
            onClick={[Function]}
            type="button"
          >
            5
          </button>
          <button
            name="6"
            onClick={[Function]}
            type="button"
          >
            6
          </button>
          <button
            name="x"
            onClick={[Function]}
            type="button"
          >
            x
          </button>
        </div>
        <div
          className="group4"
        >
          <button
            name="1"
            onClick={[Function]}
            type="button"
          >
            1
          </button>
          <button
            name="2"
            onClick={[Function]}
            type="button"
          >
            2
          </button>
          <button
            name="3"
            onClick={[Function]}
            type="button"
          >
            3
          </button>
          <button
            name="-"
            onClick={[Function]}
            type="button"
          >
            -
          </button>
        </div>
        <div
          className="group5"
        >
          <button
            className="span2"
            name="0"
            onClick={[Function]}
            type="button"
          >
            0
          </button>
          <button
            name="."
            onClick={[Function]}
            type="button"
          >
            .
          </button>
          <button
            name="="
            onClick={[Function]}
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
